src/scripts/main.js

```diff
@@ -355,6 +355,25 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in your HTML. If it doesn't, `document.querySelector('.dashboard')` will return `null`, and attempting to access `firstElementChild` on `null` will cause an error. Consider adding a check to ensure `table` is not `null` before proceeding.